web/src/components/PoisDesktop.tsx

```diff
@@ -46,6 +46,7 @@ type PoisDesktopProps = {
   switchFeature: (step: 1 | -1) => void
   selectFeature: (feature: PoiFeature | null) => void
   direction: UiDirectionType
+  showNavigation: boolean
```

Comment:
This term is really ambiguous but I don't really know what would be better. Could be avoided if you just passed the number of pois and did the check here. I think it would also make more sense from a logical perspective as the `PoisPage` should not be responsible to decide whether buttons should be shown on desktop or not. Currently I would just wonder when it would happen that we don't want to show the buttons.

Reply:
You can just pass the amount of features instead of the boolean. I am fine with keeping the boolean but it should have a self explanatory name. In the context of a map `navigation` just means something different for me.

Reply:
you are right, renamed to `showFeatureSwitch`